src/backend/mailbox_manager/utils/dimail.py

```diff
@@ -255,40 +257,54 @@ def raise_exception_for_unexpected_response(self, response):
             f"[DIMAIL] unexpected error: {response.status_code} {error_content}"
         )
 
-    def notify_mailbox_creation(self, recipient, mailbox_data, issuer=None):
+    def notify_mailbox_info(
+        self,
+        recipient,
+        mailbox_data,
+        is_new_mailbox,
+        issuer=None,
+    ):
```

Comment:
I'm not fan of the "new argument + condition" here, I would rather write another function and factorize code that can be like:

``` python
def notify_mailbox_creation(...):
    _send_mailbox_related_email(...)

def notify_mailbox_password_reset(...):
     _send_mailbox_related_email(...)

def _send_mailbox_related_email(...):
    ...
```

Reply:
I don't mind either ways so I'll do it. Would you know why you don't like it ? So I can anticipate next time :)